enterprise-suite/gotests/tests/elasticsearch/elasticsearch_test.go

```diff
@@ -0,0 +1,35 @@
+package console
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/lightbend/console-charts/enterprise-suite/gotests/testenv"
+	"github.com/lightbend/console-charts/enterprise-suite/gotests/util/lbc"
+	"github.com/lightbend/console-charts/enterprise-suite/gotests/util/urls"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestGrafana(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Elasticsearch Suite")
+}
+
+var _ = BeforeSuite(func() {
+	testenv.InitEnv()
+	installer := lbc.DefaultInstaller()
+	installer.AdditionalHelmArgs = []string{"--set enableElasticsearch=true"}
+})
+
+var _ = AfterSuite(func() {
+	testenv.CloseEnv()
+})
+
+var _ = Describe("all:elasticsearch", func() {
+	It("can access Elasticsearch", func() {
+		addr := fmt.Sprintf("%v/%v", testenv.ConsoleAddr, "service/elasticsearch/")
+		Expect(urls.Get200(addr)).To(Succeed(), addr)
```

Comment:
This is already covered by tests/nginx_test.go:47

Reply:
I removed that, because the default install doesn't install ES anymore. So decided to put it in a dedicated test here instead when ES is enabled.